content/blog/2018/07/2018-07-25-contributor-summit.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit"
+tags:
+- community
+- event
+- jenkins-world
+author: tracymiranda
+---
+
+The Jenkins Contributor summit is where the current and future contributors of the Jenkins project get together.
+This summit will be on Monday, September 17th 2018 in San Francisco, just before link:https://www.cloudbees.com/devops-world/san-francisco[DevOpsWorld | Jenkins World].
+The summit brings together community members to learn, meet and help shape the future of Jenkins. 
+In the Jenkins commmunity we value all types and sizes of contributions and love to welcome new participants.
+
+== Topics
+There are plenty of exciting developments happening in the Jenkins community. 
+The summit will feature a 'State of the Project' update including updates from the Jenkins officers. 
+We will also have updates on the 'Big 5' projects in active development:
+* Jenkins Essentials
+* Jenkins X
+* Configuration as Code
+* Jenkins Pipeline
+* Cloud Native Jenkins Architecture
+
+Plus we will also have updates from our Google Summer of Code students, and more!
+
+== Agenda
+The agenda is shaping up well and here is the outline so far.
+
+*  9:00am Kickoff & Welcome with coffee/pastries
+* 10:00am Project Updates
+* 12:00pm Lunch
+*  1.00pm BoF/Unconference

```

Comment:
It is better to explain the BoF acronym in the post

Reply:
Good idea

Reply:
Would it be possible to add Cloud Native SIG meeting in parallel with BoFs? It will be a follow-up to Cloud Native Archutecture Update. Ready to drive it

Reply:
Proposed it in https://groups.google.com/forum/#!topic/jenkins-cloud-native-sig/ddTr5A2psmE